### PR TITLE
Adds rating and extra tag, adds new status, and bump tweepy version.

### PR DESCRIPTION
**Pull request by thecodepixi**
I found that the Giphy API accepts a `rating` parameter on the `random` endpoint, so I added a `g` rating to help combat some of the _spicy_ and also _just plain weird_ GIFs that pop up sometimes. I also added the word `hugging` to the tags to broaden the potential results, and I bumped Tweepy up to the latest version `3.10.0`. 

Let me know if you want me to change anything!

### CODE REVIEW

**Comment by thecodepixi** [comment]
Oh, and I added a new status as well!

File: hug-bot.py [hunk 1]
```diff
@@ -24,16 +24,17 @@ def twitter_api():
...
     tag = ' '.join(tags)
+    rating = 'g'
     fmt = 'json'
     open(gif_filename, 'w')
     # Twitter has a max upload size of 15MB
     # TODO: tweepy has a bug and doesn't let more than ~5mb, update the max when they fix it
     # https://github.com/tweepy/tweepy/pull/1414 (hopefully?)
     while os.path.getsize(gif_filename) == 0 or os.path.getsize(gif_filename) > 5242880:
-        api_response = giphy_api.gifs_random_get(giphy_api_key, tag=tag, fmt=fmt)
+        api_response = giphy_api.gifs_random_get(giphy_api_key, rating=rating, tag=tag, fmt=fmt)
```
Comment (achasveachas) [inline hug-bot.py:37]: That's awesome!

File: statuses.py [hunk 2]
```diff
@@ -33,4 +33,5 @@
    "Hugs are great!\n\n\n\n\n\n\nⓘ Official sources all agree this is the case",
    "I'm a:\n⚪️ Man\n⚪️ Woman\n🔘 Twitter Bot\n\nLooking for a:\n⚪️ Man\n⚪️ Woman\n🔘 Hug!",
    "The snuggle is real... 🤗",
+   "Don't forget to wear your mask! It's like a hug for your face 😷\n\nAnd while you're here, have a virtual hug 🤗"
```
Comment (achasveachas) [inline statuses.py:36]: Cute :hugs:

**Review by achasveachas — COMMENTED** [review]
This looks great!

Will approve/merge once I test it manually just to be 100% sure nothing breaks.

**Review by achasveachas — APPROVED** [review]

**Pull request merged**